src/components/Input/InputWrapper.tsx

```diff
@@ -29,9 +32,20 @@ const Wrapper = styled.div<{ $error: boolean }>`
       background: ${theme.click.field.color.background.hover};
       color: ${theme.click.field.color.text.hover};
     }
-    & > input {
+    & > input, & > textarea {
```

Comment:
I think the wrapper is trying to control too much of the children. This should probably be delegated to the children so that each one can take care of themselves. We are running the risk of adding new behaviour (a new child component) and every time we change the parent (Wrapper) to support the new component, we break existing ones.

Reply:
Fixed